code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -43,6 +42,11 @@
 		synd_mind.assigned_role = "Syndicate"
 		synd_mind.special_role = "Syndicate"//So they actually have a special role/N
 		log_game("[synd_mind.key] (ckey) has been selected as a nuclear operative")
+		if(ishuman(synd_mind))//don't want operatives burning to death instantly.
+			var/mob/living/carbon/human/human = synd_mind
+			if(human.dna && human.dna.species.id == "plasmaman")

```

Comment:
no, do ```if(human.dna && human.dna.species.dangerous_existence)```, so it works for all future species and any current others that are too dangerous.